src/app/app.component.css

```diff
@@ -1,6 +1,9 @@
 prx-header {
   max-width: 100%;
 }
+prx-header >>> nav {
```

Comment:
slight regression on the recent changes to prx-header. I needed to push this dropdown menu back over to the left with a wider parent element.